config/onprem-iso-fcc.yaml

```diff
@@ -107,3 +120,16 @@ storage:
           mode: 0644
           contents:
             inline: replace-with-your-urlencoded-pull-secret
+        - path: /etc/assisted-service/startup_script.sh
+          mode: 0644
+          contents:
+            inline: |
+              #!/bin/bash
+              ips=$(hostname -I)
+              ipArr=($ips)
+              for ip in "${ipArr[@]}"
+                  do 
+                      printf "\n$ip assisted-api.local.openshift.io">>/etc/hosts
```

Comment:
I think it is missing "\n" at the end of "assisted-api.local.openshift.io". 

I tested this PR and the ignition generation failed 

````
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='assisted-api.local.openshift.io', port=8080): Max retries exceeded with url: /api/assisted-install/v1/clusters/7ba5d599-d936-435e-b173-68b420d4b87c/hosts (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9dd60adc10>: Failed to establish a new connection: [Errno -2] Name or service not known'))
````

It was because /etc/hosts on the live ISO was:

````
127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
::1         localhost localhost.localdomain localhost6 localhost6.localdomain6

192.168.122.5 assisted-api.local.openshift.io10.88.0.2  assisted-installer
````

Reply:
Thank you. Yes, I had seen that there was no newline at the end but the ignition config was getting generated correctly in the logs. I have fixed it now.